src/main/resources/i18n/messages.properties

```diff
@@ -369,6 +369,25 @@ AddMemberButton.modal.user-help=Search by first name, last name, or username.
 AddMemberButton.modal.role=Project Role
 AddMemberButton.modal.okText=Add Member
 
+# ========================================================================================== #
+# Project User Groups                                                                        #
+# ========================================================================================== #
+server.usergroups.remove-group.success={0} removed from this project.
```

Comment:
Can you change the `-` to `.` here and in the translation keys below? Just to keep with the format we have for the keys

Reply:
Updated in d2184e556cdbb0070be1b52a06b03fe38dd037a2